libs/template/templates/default-python/databricks_template_schema.json

```diff
@@ -32,10 +32,9 @@
         },
         "serverless": {
             "type": "string",
-            "default": "no",
+            "default": "yes",
```

Comment:
Could you also add something to https://github.com/databricks/cli/blob/main/NEXT_CHANGELOG.md about this change?